python/lsst/analysis/tools/actions/vector/calcBinnedCompleteness.py

```diff
@@ -0,0 +1,145 @@
+# This file is part of analysis_tools.
```

Comment:
How is this different to the one in keyedData?

Reply:
It is not; I was supposed to have deleted this one since it returns KeyedData and should not be under vector.